contracts/token/account_impl.cpp

```diff
@@ -121,6 +122,9 @@ void account_impl::add_balance(extended_asset value) {
 void account_impl::sub_deposit(extended_asset value) {
    check_account_is_valid();
    check(_this->deposit->amount >= value.quantity.amount, "overdrawn deposit");
+   check(!_st->option(token_impl::opt::floatable) ||
```

Comment:
```suggestion
   check(_st->option(token_impl::opt::floatable) ||
```
Logically, "`floatable` is true" or "calculated result of modulus needs to be zero" (because `floatable` is false).